flow/cmd/api.go

```diff
@@ -205,13 +208,27 @@ func APIMain(ctx context.Context, args *APIServerParams) error {
 
 	requestLoggingMiddleware := middleware.RequestLoggingMiddleWare()
 
-	// Interceptors are executed in the order they are passed to, so unauthorized requests are not logged
-	interceptors := grpc.ChainUnaryInterceptor(
-		authGrpcMiddleware,
-		requestLoggingMiddleware,
-	)
+	serverOptions := []grpc.ServerOption{
+		// Interceptors are executed in the order they are passed to, so unauthorized requests are not logged
+		grpc.ChainUnaryInterceptor(
+			authGrpcMiddleware,
+			requestLoggingMiddleware,
+		),
+	}
+
+	var otelManager *otel_metrics.OtelManager
+	if args.EnableOtelMetrics {
+		otelManager, err = otel_metrics.NewOtelManager(otel_metrics.FlowApiServiceName)
+		if err != nil {
+			return fmt.Errorf("unable to create otel manager: %w", err)
+		}
+		_ = otelManager.Meter
```

Comment:
?

Reply:
was testing smth locally, this chunk got added